charts/nextcloud/templates/deployment.yaml

```diff
@@ -329,17 +329,18 @@ spec:
       {{- if .Values.nextcloud.extraVolumes }}
 {{ toYaml .Values.nextcloud.extraVolumes | indent 6 }}
       {{- end }}
+      # this is the securityContext for ALL containers in the nextcloud pod
       securityContext:
       {{- if .Values.nextcloud.podSecurityContext }}
         {{- with .Values.nextcloud.podSecurityContext }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
       {{- else }}
-        {{- if .Values.nginx.enabled }}
-        # Will mount configuration files as www-data (id: 82) for nextcloud
+        {{- if eq .Values.image.flavor "fpm-alpine" }}
```

Comment:
I know it's not great, but could we match on substring "alpine"?

Reply:
I also never used regex in a helm/go template before, so I can't really help :woman_shrugging: